test/cim/iec61970/base/wires/test_power_transformer.py

```diff
@@ -3,14 +3,16 @@
 #  This Source Code Form is subject to the terms of the Mozilla Public
 #  License, v. 2.0. If a copy of the MPL was not distributed with this
 #  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+from _pytest.python_api import raises
```

Comment:
Should just be:
```from pytest import raises```

Reply:
Replaced all occurrences of `_pytest.python_api` with `pytest` (5 total).